### PR TITLE
py-dockerpty: fix dependencies

### PR DESCRIPTION
**Pull request by emcrisostomo**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.6 18G3020
Xcode 11.3.1 11C504

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by emcrisostomo** [comment]
Added `py-six` to the list of dependencies, as specified in the package descriptors

File: python/py-dockerpty/Portfile [hunk 1]
```diff
@@ -22,14 +22,15 @@ checksums           rmd160  6d0604c549d301b92b98c2b25825ff04881e48cd \
...
+python.versions     27 35 36 37 38
 
 if {${subport} ne ${name}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
     depends_lib-append \
-                    port:py${python.version}-docker
+        port:py${python.version}-docker \
+        port:py${python.version}-six
```
Comment (reneeotten) [inline python/py-dockerpty/Portfile]: why not keep these lines aligned with the rest of the Portfile as it was before?

Comment (emcrisostomo) [reply python/py-dockerpty/Portfile]: You're right @reneeotten, no reason but Emacs deciding otherwise and my not realising. Fixed.

**Comment by reneeotten** [comment]
you are also adding subports for PY37 and PY38, which should ideally be mentioned in the commit message. Did you verify the package works with these new versions?

**Comment by emcrisostomo** [comment]
Thanks @reneeotten, I'll amend the commit message as soon as the checks are finished. 3.7 has been checked with the other docker ports I'm pushing, while 3.8 has been tested with a custom python program (docker-compose won't build yet with 3.8 because of unavailable dependencies, and that's why I'm doing it one by one).

**Comment by emcrisostomo** [comment]
Travis CI builds failed for timeout. Merging.

**Pull request merged**